ocaml/idl/datamodel.ml

```diff
@@ -954,6 +954,17 @@ module PIF = struct
              Api_errors.pif_has_fcoe_sr_in_use]
       ()
 
+  let set_disallow_unplug = call
+    ~name:"set_disallow_unplug"
+    ~doc:"Set whether unplugging the PIF is allowed"
+    ~hide_from_docs:false
+    ~in_oss_since:None
+    ~in_product_since:rel_orlando
+    ~params: [ Ref _pif, "self", "Reference to the object"
+             ; Bool, "value", "New value to set" ]
+    ~allowed_roles:_R_POOL_OP
+    ()
+
```

Comment:
I assume this allowed role is fine?

Reply:
If that was the original role than yes.

Reply:
I've looked at the definitions of `let create_obj` and `let field`, and it seems that the field setter role will default to `~messages_default_allowed_roles`, if I'm not mistaken, which is this, so this is perfect.